pandas/core/indexes/interval.py

```diff
@@ -845,7 +845,7 @@ def _convert_list_indexer(self, keyarr):
 
         # we have missing values
         if (locs == -1).any():
-            raise KeyError
+            raise KeyError(keyarr[locs == -1].tolist())
```

Comment:
is this user faing?

Reply:
Yep, non existing Keys for IntervalIndex

Reply:
@jreback i think this is about ready, only question is if we want to further flesh out the exception message here